core/src/View/Kokkos_ViewAlloc.hpp

```diff
@@ -188,6 +188,54 @@ struct ViewValueFunctor {
   }
 };
 
+template <class DeviceType, class ValueType>
+struct ViewValueFunctorHostSerialInit {
+  using ExecSpace = typename DeviceType::execution_space;
+
+  ValueType* ptr;
+  size_t n;
+
+  ViewValueFunctorHostSerialInit() = default;
+  ViewValueFunctorHostSerialInit(const ViewValueFunctorHostSerialInit&) =
+      default;
+  ViewValueFunctorHostSerialInit& operator=(
+      const ViewValueFunctorHostSerialInit&) = default;
```

Comment:
Any reason to declare (only) some of the special member functions explicitly?
```suggestion
```

Reply:
Removed